docs/scala_toolchain.md

```diff
@@ -78,7 +78,7 @@ scala_register_toolchains()
     declare_deps_provider(
         name = "my_scala_xml_provider",
         deps_id = "scala_xml",
-        deps = ["@scala_xml_dep"],
+        deps = ["@io_bazel_rules_scala_scala_xml"],
```

Comment:
`@io_bazel_rules_scala_scala_xml` is internal to Rules Scala codebase. The whole point of defining custom deps provider is to be able to use deps from your workspace. `@scala_xml_dep` is non existing name on purpose, it should come from users workspace and may have different names depending on which external deps loader it uses.

Reply:
Ah I see - I just couldn't find how to define it or where it is coming from... I will close the PR